persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -9,9 +18,99 @@ CREATE TABLE IF NOT EXISTS users
     first_name      VARCHAR(36) NOT NULL,
     last_name       VARCHAR(36) NOT NULL,
     privilege_level VARCHAR(16) NOT NULL,
-    password_hash   BYTEA       NOT NULL
+    country_id      BIGINT,
+    password_hash   BYTEA       NOT NULL,
+
+    CONSTRAINT fk_user_country FOREIGN KEY (country_id) REFERENCES countries (id)
+);
+
+CREATE TABLE IF NOT EXISTS schools
+(
+    id         BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP,
+    name       VARCHAR(36) NOT NULL,
+    address    VARCHAR(64) NOT NULL,
+    hidden     BOOLEAN              DEFAULT FALSE,
```

Comment:
❓ What does hidden mean?

Reply:
if Amanda wanted to "disable" a school for any reason (e.g. temporarily unavailable due to natural disaster)?